src/goto-checker/report_util.cpp

```diff
@@ -198,7 +198,11 @@ get_sorted_properties(const propertiest &properties)
       const auto &p2 = pit2->second.pc->source_location;
       if(p1.get_file() != p2.get_file())
         return id2string(p1.get_file()) < id2string(p2.get_file());
-      else if(!p1.get_line().empty() && !p2.get_line().empty())
```

Comment:
Github won't let me comment on the correct lines, but a few lines above this there is a comment that describes the ordering - I think that needs updated to reflect your changes.

Reply:
Done.